components/compositing/compositor.rs

```diff
@@ -1847,6 +2102,10 @@ impl<Window: WindowMethods + ?Sized> IOCompositor<Window> {
                     // and check if it is the one layout is expecting,
                     let epoch = Epoch(epoch);
                     if *pending_epoch != epoch {
+                        warn!(
```

Comment:
Is this really a warning case or is this something that we might expect in the normal execution of the program? If it's something normal, `debug!()` or `trace!()` is a better option.

Reply:
i’m genuinely not sure, do you know? all i know for now is that it doesn’t really happen outside of some multiview-related fault scenarios, like failing to update the frame tree or having too many webviews open (see limitations), which makes me lean towards this shouldn’t happen.